reporter-web-app/src/models/WebAppOrtResult.js

```diff
@@ -512,6 +512,10 @@ class WebAppOrtResult {
         return false;
     }
 
+    hasLabels() {
```

Comment:
function seem to be sorted, should we move `hasLabels() further down?

Reply:
Moved it to the right location, good catch.